internal/cli/atlas/deployments/diagnostics.go

```diff
@@ -121,7 +113,7 @@ func DiagnosticsBuilder() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVarP(&opts.Output, flag.Output, flag.OutputShort, "", usage.FormatOut)
+	cmd.Flags().StringVarP(&opts.Output, flag.Output, flag.OutputShort, "json", usage.FormatOut)
```

Comment:
making it json by default given this is what we expect to be reported, question for reviewers, should we remove the flag entirely and keep it always json?

Reply:
@tibulca @blva what is your take on it?